src/styles/header.scss

```diff
@@ -0,0 +1,72 @@
+.header {
+  margin: 0 auto;
+  width: 100%;
+  background-color: $bg-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  position: sticky;
+  z-index: 1000;
+
+  &__container {
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 50px;
+  }
+
+  &__logo-link {
+    display: block;
+    height: 40px;
```

Comment:
The `.header__logo-link` class has a `height: 40px;` property. This might not be needed if the height of the header is controlled by the padding of the `.header__container`. Consider removing it.